stdlib/public/core/Set.swift

```diff
@@ -1123,16 +1130,11 @@ extension Set {
   ///   otherwise, `false`.
   @inlinable
   public func isDisjoint(with other: Set<Element>) -> Bool {
-    return _isDisjoint(with: other)
-  }
-    
-  @inlinable
-  internal func _isDisjoint<S: Sequence>(with other: S) -> Bool
```

Comment:
This function is ABI, so we can't remove it. I think it makes sense to keep it (and the public sequence-based entry point) intact, and only change the specialized overload.

Reply:
Ah sure, makes sense. Thanks

Reply:
Just fixed :)